pilot/lib/pilot/build_manager.rb

```diff
@@ -81,7 +79,12 @@ def list(options)
         config[:app_identifier] = UI.input("App Identifier: ")
       end
 
-      builds = app.all_processing_builds + app.builds
+      if config[:app_platform].to_s.length == 0
+        config[:app_platform] = ask("App Platform (ios, appletvos, osx): ")
+      end
```

Comment:
Seems weird to put this in the `list` method, since you need the platform for commands like `builds`, `distribute` etc as well.